awx/settings/local_settings.py.docker_compose

```diff
@@ -276,3 +276,5 @@ TEST_OPENSTACK_PROJECT = ''
 # Azure credentials.
 TEST_AZURE_USERNAME = ''
 TEST_AZURE_KEY_DATA = ''
+
+AWX_ANSIBLE_COLLECTIONS_PATHS = ['/tmp/collections']
```

Comment:
this still doesn't exist by default, that doesn't break job runs in the dev environment, right?